src/pollution_model/run_medusa_2.py

```diff
@@ -0,0 +1,520 @@
+"""
+This script takes in appropriate datasets as input and runs the MEDUSA2.0 model to calculate
+TSS (total suspended solids), TCu (total copper), DCu  (dissolved copper), TZn (total zinc), and DZn (dissolved zinc).
+
+DOI of the model paper: https://doi.org/10.3390/w12040969
+"""
+
+import logging
+import math
+from enum import StrEnum
+
+import geopandas as gpd
+import pandas as pd
+from sqlalchemy.engine import Engine
+from sqlalchemy.ext.declarative import declarative_base
+
+from src.digitaltwin import setup_environment
+from src.digitaltwin.tables import create_table
+from src.digitaltwin.utils import LogLevel, setup_logging, get_catchment_area
+from src.flood_model.serve_model import create_building_database_views_if_not_exists
+from src.pollution_model.pollution_tables import MEDUSA2ModelOutput
+
+log = logging.getLogger(__name__)
+
+Base = declarative_base()
+
+
+# Enum strings are assigned as they are described in the original paper
+class SurfaceType(StrEnum):
+    """
+    StrEnum to represent the surface types that a feature could have.
+
+    These include:
+        Concrete Roof = "Cr",
+        Copper Roof = "Cu",
+        Galvanised Roof = "Gv",
+        Asphalt Road = "Rd",
+        Car Park = "CrP".
+    """
```

Comment:
Please update the docstring to include an `Attributes` section, you can refer to the `rainfall_enum.py` script for consistency.